packages/scrollable-video/src/components/video.js

```diff
@@ -185,7 +185,7 @@ const ForwardRefVideo = React.forwardRef(
           true /* `autoPlay` is required by iOS. If `autoPlay` is not true, and then `loadedData` event won't be triggered */
```

Comment:
If we listen to canplay event, would autoPlay also be required by iOS ?

If so, could we update the comment `loadedData` to `canplay` ?

Reply:
[df4512a](https://github.com/twreporter/orangutan-monorepo/pull/130/commits/df4512af22d8f2d178cce90bca1d5ed5aa26d452) removes `autoPlay` attribute.

Reply:
[df4512a](https://github.com/twreporter/orangutan-monorepo/pull/130/commits/df4512af22d8f2d178cce90bca1d5ed5aa26d452) fixes it.

Reply:
Yes it surely does 👍

My last question: 
The fix adds `key` prop to recreate `videoRef` when video source changes.
But I am still curious about why this corner case occurs in this change ?

Before [391928d](https://github.com/twreporter/orangutan-monorepo/pull/130/commits/391928dcc0c6ee2967867a865d852c3ef8cdbb72) and this change, the <Video> did not have `key` prop hence the `videoRef` didn't be recreated and no new ScrollTrigger instances were created by `useScrollTrigger` except for the first one. But why the old ways didn't suffer from this issue ? I still don't get it...